_docs/layout/page-layouts.md

```diff
@@ -16,35 +16,27 @@ resource: true
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
 ### Base layouts
-- Use the account navigation bar to navigate between applications,
-  manage roles, and manage preferences on both the user and the account
-  level.
+- Use the 'Utility nav' to navigate between applications, manage roles, and manage preferences on both the user and the account level.
 
-- Use the main navigation to navigate between the main sections of the
-  application. Make sure your application consistently
-  uses either a top navigation or a side navigation, but never both. You can
-  use dropdown for subcategories, but be sure to only use one sub-level.
+- Use the main navigation to navigate between the main sections of the application. Make sure your application consistently uses either a top navigation or a side navigation, but never both. You can use dropdown for subcategories, but be sure to only use one sub-level.
```

Comment:
The images associated with this section identifies side navigation as "left nav". Is it always on the left?

Reply:
Just a nit, but I feel like we're repeating the word navigation a lot. I would suggest removing "main navigation" and revising to this: "Use either 'Top nav' or 'Left nav', to navigate between the main sections of the application. Make sure your application uses either top nav or left nav, but never both. Use dropdown menus for subcategories, but be sure to use only one sub-level."

Reply:
I `%s/side nav/left nav/g` because it is always on the left and this is how it is referred to within the component library.